core/modules/file/file.install

```diff
@@ -1199,3 +1201,11 @@ function file_update_1008() {
   }
   db_create_table('cache_entity_file', $table);
 }
+
+/** 
+ * Remove the file_type_determine queue.
+ */
+function file_update_1009() {
+  $queue = BackdropQueue::get('file_type_determine');
```

Comment:
If there are still items in the queue, then the batch needs to be run to finish it:

```suggestion
  $queue = BackdropQueue::get('file_type_determine');
  if ($queue->numberOfItems() > 0) {
    file_needs_type_classification(TRUE);
  }
```

Reply:
Of course. Added!